### ISSUE TITLE
Filter event spam

### ISSUE THREAD

**Issue by 2Cubed** [CactusDev/Cereus#35]
If an event occurs too quickly, it should be discounted. (For example, if someone follows and unfollows multiple times within the same day, we should not continually notify chat.)

**Comment by Innectic** [comment]
This should be cached in redis using redis TTL, so that the event expires after like 500 seconds (or whatever) and can happen again

**Comment by 2Cubed** [comment]
@Innectic Agreed. We could consider potentially adding a microservice around Redis in the future, too, if we end up using it enough.

**Comment by Innectic** [comment]
@2Cubed Sepal has to exist for repeats still, so we can make a cache packet

**Comment by Alkali-Metal** [comment]
If possible, having a different "cooldown" for different events would probably be beneficial, as sometimes events might happen more frequently than other times and you want it to announce at different intervals. (Should maybe be configurable per bot?)
